pkg/cloudprovider/launchtemplate.go

```diff
@@ -209,7 +210,7 @@ func (p *LaunchTemplateProvider) createLaunchTemplate(ctx context.Context, optio
 	if err != nil {
 		return nil, err
 	}
-	logging.FromContext(ctx).Debugf("Created launch template, %s", *output.LaunchTemplate.LaunchTemplateName)
+	logging.FromContext(ctx).Debugf("created launch template")
```

Comment:
Can we add the launch template id here?

Reply:
or above, even

Reply:
I couldn't add it above as the template needed to be created to get the template-id.